### PR TITLE
Set up Associations

### PR DESCRIPTION
**Pull request by bruk19**
In this PR:-

- Created a class `Classroom`
- Added `@label` instance variable, and initialized in the constructor.
- Added Setter and getter for `@label`
- Created the `has-many` side
- Created the `belongs-to` side
- Created a class `Book`
- Added `@title` and `@author` instance variables, and initialized in the constructor.
- Setters and getters for instance variables
- Created a class `Rental`
- Added `@date` instance variable, and initialized in the constructor.
- Setter and getter for `@date`
- Created the `has-many` side of `Book` and `Rental`
- Created the `belongs-to` side of `Rental` and `Book`
- Created the `has-many` side of `Person` and `Rental`
- Created the `belongs-to` side of `Rental` and `Person`
- Modify the constructor of `Rental`

### CODE REVIEW

File: Classroom.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+  def initialize(label)
+    @label = label
+    @stunet = []
+  end
+
+  # Method to add student one by one
+  def add_student(student)
+    @student.push(student)
+  end
```
Comment (ShahierNashaat) [inline Classroom.rb]: - [ ] Kindly be noticed that you should also asign the classroom to the student that was sent in the method parameter.

```ruby
 def add_student(student)
    @student.push(student)
    student.classroom = self
 end
```

File: student.rb [hunk 2]
```diff
@@ -11,4 +11,10 @@ def play_hooky
     return "¯\(ツ)/¯"
   end
```
Comment (ShahierNashaat) [inline student.rb:15]: - [ ] Kindly make the `@classroom` attribute reachable.

```ruby
attr_reader :classroom
```

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @bruk19, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @bruk19 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**